cmd/minikube/cmd/start.go

```diff
@@ -1766,14 +1799,19 @@ $ minikube config unset kubernetes-version`)
 
 	kubernetesSemver := strings.TrimPrefix(strings.ToLower(paramVersion), version.VersionPrefix)
 	if isTwoDigitSemver(kubernetesSemver) {
-		kubernetesSemver = getLatestPatch(kubernetesSemver)
+		potentialPatch := getLatestPatch(kubernetesSemver)
+		if potentialPatch != `` {
+			kubernetesSemver = potentialPatch
+		} else {
+			return potentialPatch, ErrPatchNotFound
+		}
 	}
 	nvs, err := semver.Make(kubernetesSemver)
 	if err != nil {
```

Comment:
Not sure if here we want to send back an error here instead of just exiting. With the error return value it could  be better to now return an error and handle it outside getKubernetesVersion

Reply:
good point and good catch ! I think since it is already been implemented like that we can leave this one alone, and if we want we can refactor it in a follow up PR (for sake of keeping this PR small)